### PR TITLE
Add label arg to apply_inverse

### PR DESCRIPTION
**Pull request by teonlamont**
Addresses #1797.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
The Travis builds are angry, did it work on your end?

File: mne/minimum_norm/tests/test_inverse.py [hunk 1]
```diff
@@ -234,6 +234,16 @@ def test_apply_inverse_operator():
     assert_true(stc.data.max() < 35)
     assert_true(stc.data.mean() > 0.1)
 
+    # test without using a label (so delayed computation is used)
+    label = read_label(fname_label % 'Aud-lh')
+    stc = apply_inverse(evoked, inv_op, lambda2, "MNE")
+    stc_label = apply_inverse(evoked, inv_op, lambda2, "MNE",
+                              label=label)
+    assert_true(stc_label.subject == 'sample')
```
Comment (Eric89GXL) [inline mne/minimum_norm/tests/test_inverse.py]: FYI you can use `assert_equal`

Comment (agramfort) [inline mne/minimum_norm/tests/test_inverse.py]: +1

**Comment by agramfort** [comment]
besides LGTM

maybe see why travis not happy

**Comment by teonlamont** [comment]
in the test, i'm getting a dimensionality mismatch for some reason. is there a way to download the sample data that is being used on travis. it's differs from the sample_data with its trunc files.

**Comment by agramfort** [comment]
Yes see Travis yaml file

**Comment by teonlamont** [comment]
:thumbsup: 
fix the problem, it was a matter of how the vertices were being extracted.

**Comment by agramfort** [comment]
LGTM

+1 for merge if travis is happy

**Pull request merged**